src/plugins/console/public/application/components/storage_quota_error.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiButton, EuiButtonEmpty } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+interface Props {
+  onClearHistory: () => void;
+  onDisableSavingToHistory: () => void;
+}
+
+export const StorageQuotaError = ({ onClearHistory, onDisableSavingToHistory }: Props) => (
```

Comment:
Could we change these handlers to also hide the toast? Otherwise there isn't a clear feedback of what happened to the user

https://user-images.githubusercontent.com/1191206/142869771-56064f92-6ca0-4260-94be-cd6721b634e6.mov